frontend/ui/font.lua

```diff
@@ -10,30 +12,30 @@ local logger = require("logger")
 local Font = {
     fontmap = {
         -- default font for menu contents
-        cfont = "noto/NotoSans-Regular.ttf",
+        cfont = "NotoSans-Regular.ttf",
```

Comment:
Isn't losing the subdirectory an issue on !Android, where they do live in a noto subdirectory?

Reply:
The fonts are scanned in ./fonts/* first. :-)

Reply:
Oh drat, my reversal commit introduced an issue here sometimes skipping that scan. Meh.

Reply:
@NiLuJe The logic could be slightly different, i.e. try with full path and if fail use [util.spiltFilePathName](http://koreader.rocks/doc/modules/util.html#splitFilePathName) to search all folders. It'd be negligibly faster on platforms that ship all fonts; it's more the future readability of the logic that I'm trying to figure out.